The SVE kernels will be used automatically only on CPUs which support them, right?  Julia runs fine on CPUs with SVE, see https://doi.org/10.1109/CLUSTER51413.2022.00072 and https://github.com/giordano/julia-on-gh200/ for a couple of examples.

---

Separately, [aarch64-linux-musl is failing](https://buildkite.com/julialang/yggdrasil/builds/28810#019db047-ca31-4cc9-83d2-8cc4d1be0fd6/L6606)
```
[13:56:12] In file included from /workspace/srcdir/fftw-3.3.11/kernel/timer.c:29:
[13:56:12] /workspace/srcdir/fftw-3.3.11/kernel/cycle.h:543:18: error: conflicting types for 'ticks'
[13:56:12]  typedef uint64_t ticks;
[13:56:12]                   ^~~~~
[13:56:12] /workspace/srcdir/fftw-3.3.11/kernel/cycle.h:442:25: note: previous declaration of 'ticks' was here
[13:56:12]  typedef struct timespec ticks;
[13:56:12]                          ^~~~~
[13:56:12] /workspace/srcdir/fftw-3.3.11/kernel/cycle.h:544:21: error: conflicting types for 'getticks'
[13:56:12]  static inline ticks getticks(void)
[13:56:12]                      ^~~~~~~~
[13:56:12] /workspace/srcdir/fftw-3.3.11/kernel/cycle.h:444:21: note: previous definition of 'getticks' was here
[13:56:12]  static inline ticks getticks(void)
[13:56:12]                      ^~~~~~~~
[13:56:12] /workspace/srcdir/fftw-3.3.11/kernel/cycle.h:92:47: error: conflicting types for 'elapsed'
[13:56:12]  #define INLINE_ELAPSED(INL) static INL double elapsed(ticks t1, ticks t0) \
[13:56:12]                                                ^~~~~~~
[13:56:12] /workspace/srcdir/fftw-3.3.11/kernel/cycle.h:550:1: note: in expansion of macro 'INLINE_ELAPSED'
[13:56:12]  INLINE_ELAPSED(inline)
[13:56:12]  ^~~~~~~~~~~~~~
[13:56:12] /workspace/srcdir/fftw-3.3.11/kernel/cycle.h:451:22: note: previous definition of 'elapsed' was here
[13:56:12]  static inline double elapsed(ticks t1, ticks t0)
[13:56:12]                       ^~~~~~~
```
Not quite sure what's the issue, sounds like an upstream bug?

--- a/F/FFTW/build_tarballs.jl
+++ b/F/FFTW/build_tarballs.jl
@@ -38,8 +38,8 @@ fi
 #    FLAGS+=( --enable-avx512 );
 # fi
 
-# Enable NEON on Aarch64
-if [[ "${target}" == aarch64-* ]]; then FLAGS+=( --enable-neon ); fi
+# Enable NEON and SVE on Aarch64
+if [[ "${target}" == aarch64-* ]]; then FLAGS+=( --enable-neon --enable-sve ); fi
 
 # Julia-Aarch64 platforms (MacOS, Linux, FreeBSD) nowadays give access to
 # CNTVCT_EL0 cycle counter in userspace.  (We need to enable this explicitly

--- a/F/FFTW/build_tarballs.jl
+++ b/F/FFTW/build_tarballs.jl
@@ -38,8 +38,8 @@ fi
 #    FLAGS+=( --enable-avx512 );
 # fi
 
-# Enable NEON and SVE on Aarch64
-if [[ "${target}" == aarch64-* ]]; then FLAGS+=( --enable-neon --enable-sve ); fi
+# Enable NEON on Aarch64
+if [[ "${target}" == aarch64-* ]]; then FLAGS+=( --enable-neon ); fi
 
 # Julia-Aarch64 platforms (MacOS, Linux, FreeBSD) nowadays give access to
 # CNTVCT_EL0 cycle counter in userspace.  (We need to enable this explicitly

--- a/F/FFTW/build_tarballs.jl
+++ b/F/FFTW/build_tarballs.jl
@@ -54,6 +54,9 @@ if [[ "${target}" == i686-w64-* ]]; then FLAGS+=( --with-incoming-stack-boundary
 # On ppc64le, enable VSX
 if [[ "${target}" == powerpc64le-*  ]]; then FLAGS+=( --enable-vsx ); fi
 
+# work around https://github.com/FFTW/fftw3/issues/403
+perl -pi -e "s/CLOCK_SGI_CYCLE/CLOCK_SGI_CYCLE_NOPE/" ${WORKSPACE}/srcdir/fftw*/kernel/cycle.h
+
 # We need to do this a couple times, so functionalize it
 build_fftw()
 {

